Unix/tests/cli/test_cli.cpp

```diff
@@ -1092,19 +1158,50 @@ NitsTestWithSetup(TestOMICLI26_InvokeWsman, TestCliSetupWsman)
 }
 NitsEndTest
 
-NitsTestWithSetup(TestOMICLI27_EnumerateWsman, TestCliSetupWsman)
+NitsTestWithSetup(TestOMICLI26_InvokeWsmanSync, TestCliSetupWsman)
 {
     NitsDisableFaultSim;
 
     string out;
     string err;
     NitsCompare(
-        Exec(MI_T("omicli ei --hostname localhost -u test -p password --httpport 5985 --httpsport 5986 root/cimv2 MSFT_President"),
+        Exec(MI_T("omicli iv -synchronous --hostname localhost -u test -p password --httpport 5985 --httpsport 5986 test/cpp { X_SmallNumber } SpellNumber { num 123 }"),
              out, err), 0, MI_T("Omicli error")); 
 
     string expect;
-    NitsCompare(InhaleTestFile("TestOMICLI27.txt", expect), true, MI_T("Inhale failure"));
-    NitsCompareString(out.c_str(), expect.c_str(), MI_T("Output mismatch"));
+    NitsCompare(InhaleTestFile("TestOMICLI26.txt", expect), true, MI_T("Inhale failure"));
```

Comment:
there is no point inhaling if you are not comparing! I thing you should be using your WordCount